### PR TITLE
Field content value alerts

### PR DESCRIPTION
**Pull request by lennartkoopmann**
Depending on Graylog2/graylog2-server#1160.

### CODE REVIEW

File: app/views/partials/alerts/form_field_string_value.scala.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@(stream: org.graylog2.restclient.models.Stream, conditionDecorator: views.helpers.FieldStringValueAlertConditionDecorator)
+
+<form action="@conditionDecorator.getFormAction(stream.getId)" method="POST" class="alert-type-form alert-type-form-field-string-value" id="@conditionDecorator.getFormId">
+    <div class="well well-small">
+        <h2 style="margin-bottom: 10px;">
+            <i class="fa fa-bell-alt"></i>
+            @conditionDecorator.getFormTitle
```
Comment (dennisoelkers) [inline app/views/partials/alerts/form_field_string_value.scala.html]: Adding the title to the inline form is inconsistent with the other alert condition types.

File: conf/routes [hunk 2]
```diff
@@ -49,6 +49,7 @@ GET           /streams/:stream_id/outputs
 GET           /streams/:stream_id/alerts                                                         @controllers.AlertsController.index(stream_id: String)
 POST          /streams/:stream_id/alerts/conditions/add/messagecount                             @controllers.AlertsController.addTypeMessageCount(stream_id: String)
 POST          /streams/:stream_id/alerts/conditions/add/fieldvalue                               @controllers.AlertsController.addTypeFieldValue(stream_id: String)
+POST          /streams/:stream_id/alerts/conditions/add/fielstringdvalue                         @controllers.AlertsController.addTypeFieldStringValue(stream_id: String)
```
Comment (dennisoelkers) [inline conf/routes]: The 'd' was trying to run away but only made it six characters to the right yet! Quick, catch it!

File: app/views/partials/alerts/form_field_string_value.scala.html (hunk 1 above)
Comment (lennartkoopmann) [inline app/views/partials/alerts/form_field_string_value.scala.html]: Thx! fixed in b13b12b

File: conf/routes (hunk 2 above)
Comment (lennartkoopmann) [inline conf/routes]: haha, good catch!



fixed in bbd0e2e :)

**Comment by lennartkoopmann** [comment]
This should be okay now. Thanks for the review!

(Merging still waits for Graylog2/graylog2-server#1160 to be finished, too.)

File: conf/routes [hunk 3]
```diff
@@ -49,6 +49,7 @@ GET           /streams/:stream_id/outputs
 GET           /streams/:stream_id/alerts                                                         @controllers.AlertsController.index(stream_id: String)
 POST          /streams/:stream_id/alerts/conditions/add/messagecount                             @controllers.AlertsController.addTypeMessageCount(stream_id: String)
 POST          /streams/:stream_id/alerts/conditions/add/fieldvalue                               @controllers.AlertsController.addTypeFieldValue(stream_id: String)
+POST          /streams/:stream_id/alerts/conditions/add/fieldstringvalue                         @controllers.AlertsController.addTypeFieldStringValue(stream_id: String)
```
Comment (joschi) [inline conf/routes]: The path should be named the same as the condition: `/streams/:stream_id/alerts/conditions/add/fieldcontentvalue`

**Comment by razvanphp** [comment]
@lennartkoopmann Is this working for integer comparison, so something like "level must be smaller than 4"? or " response_status between 500 and 599 "?

**Comment by bernd** [comment]
> Is this working for integer comparison, so something like "level must be smaller than 4"? or " response_status between 500 and 599 "?

No, this only works for simple field content conditions like "field x == value y". You can use a "Field value condition" to alert on min/max etc.

**Pull request closed without merging**